### PR TITLE
update babel to 7.1.*

### PR DESCRIPTION
**Pull request by BenDiuguid**
I was getting a warning for the babel-core peer deps in another project, so I went ahead and updated the babel deps, and everything still worked for me 👍 

Let me know if there's anything else you need in this PR 😄

### CODE REVIEW

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
@izaakschroeder LGTM, shall we do a release?

File: package.json [hunk 1]
```diff
@@ -11,18 +11,18 @@
...
-    "@babel/core": "^7.0.0-beta.37"
+    "@babel/core": "^7.1.2"
   },
   "dependencies": {
-    "@babel/traverse": "^7.0.0-beta.37",
-    "@babel/types": "^7.0.0-beta.37",
-    "babylon": "^7.0.0-beta.37"
+    "@babel/traverse": "^7.1.4",
+    "@babel/types": "^7.1.3",
+    "babylon": "^7.0.0-beta.47"
```
Comment (evocateur) [inline package.json]: `babylon` should be `@babel/parser`, as the package was renamed during beta.
https://babeljs.io/docs/en/v7-migration-api#babel-parser-known-as-babylon

Comment (BenDiuguid) [reply package.json]: Sorry for the slow response, but it should be updated now!

**Review by evocateur — APPROVED** [review]

**Comment by milesj** [comment]
@izaakschroeder Any possibility of this being released?

**Comment by izaakschroeder** [comment]
Ack sorry yes @milesj I need to fix GH notifications 😭 on this right now!

**Comment by izaakschroeder** [comment]
Out in `+ babel-literal-to-ast@2.1.0` 🎉 @milesj @evocateur @ndelangen

**Pull request merged**